CONTRIBUTING.md

```diff
@@ -321,3 +321,38 @@ stripped with no prior warning or consent from the member in question.
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+## Point Release Procedure
+
+*   3 Stages
+    *   **Strategy Discovery**
+        *   Develop long term priorities, strategy and roadmap for the SDK
+        *   Release committee not yet defined as there is already a roadmap that can be used for the time being
```

Comment:
What does release committee have to do with strategy discovery?

Reply:
Ditto